tegg/core/agent-tracing/src/types.ts

```
@@ -0,0 +1,169 @@
+
+export const RunStatus = {
+  START: 'start',
+  END: 'end',
+  ERROR: 'error',
+} as const;
+export type RunStatus = (typeof RunStatus)[keyof typeof RunStatus];
+
+/** Internal config used by TracingService */
+export interface AgentTracingConfig {
+  // Reserved for future configuration options
+}
```

Biome (2.4.4)

[error] An empty interface is equivalent to {}.

(lint/suspicious/noEmptyInterface)